sdk/identity/identity/src/credentials/chainedTokenCredential.ts

```diff
@@ -56,8 +59,17 @@ export class ChainedTokenCredential implements TokenCredential {
       }
     }
 
-    if (!token && errors.length > 0) {
-      const err = new AggregateAuthenticationError(errors);
+    if (errors.length > 0) {
```

Comment:
why bother doing a length check?

Reply:
We should handle this "errors" only if its length greater than zero.